Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -118,15 +113,19 @@ export RGW_POOL_PREFIX=default
 
 ## Commands on the K8s consumer cluster
 
```

Comment:
How about adding this?
```
To install with helm, see the below section [Helm installation](#helm-installation). Otherwise, continue with the manifests installation.
```

Reply:
Added it in top so no need two paragraph